sdk/attestation/attestation/src/attestationAdministrationClient.ts

```diff
@@ -127,11 +127,11 @@ export class AttestationAdministrationClient {
 
       // The attestation token returned from the service has a PolicyResult 
       // object as the body. 
-      let token = new AttestationToken(getPolicyResult.token);
+      const token = new AttestationToken(getPolicyResult.token);
 
       // Deserialize the PolicyResult object to retrieve the underlying policy
       //  token
-      let policyResult = PolicyResult.create(token.get_body());
+      let policyResult = PolicyResult.create(token.getBody());
```

Comment:
there is a bunch of `let`s here that needs to be converted to `const`.

Reply:
Any ideas why lint doesn't find them?

Reply:
Good question, let me check.

Reply:
Agh, because the linting command lints everything except the `src` directory. I will open a PR with linting fixes.